rustls/src/conn.rs

```diff
@@ -840,16 +840,15 @@ impl CommonState {
             has_seen_eof: false,
             received_middlebox_ccs: 0,
             peer_certificates: None,
-            message_fragmenter: MessageFragmenter::new(max_fragment_size)
-                .map_err(|_| Error::BadMaxFragmentSize)?,
+            message_fragmenter: Default::default(),
```

Comment:
Nit: would prefer to spell this as `MessageFragmenter::default()`.

Reply:
Done.